resource/WebGui/app/service/views/InMoov2Gui.html

```diff
@@ -73,15 +73,53 @@
                 <div id="leftLegGuiDisplay" ng-show="service.isLeftLegActivated" class="GuiDisplays preventGuiDragSelect">
                     <img src="../img/InMoov2/legL_Activ.png"/>
                 </div>
-                <div id="batteryGuiDisplay" class="GuiDisplays preventGuiDragSelect" ng-click="i01.mouth.speak(str(Runtime.getBatteryLevel())+'percent')">
+                <div id="batteryGuiDisplay" class="GuiDisplays preventGuiDragSelect" ng-click="i01.speakBlocking(str(Runtime.getBatteryLevel())+'percent')">
                     <img src="../img/InMoov2/battery_Activ.png"/>
```

Comment:
in general you never want to refer to InMoov or InMoov2 with "i01" ... 
that was the instance name, the first one i came up with - its not supposed to be in "code"
It was only the first thing I thought of ... like clock01 = Runtime.start('clock01','Clock')
I think you wanted msg.speakBlocking( ...  anyway

Reply:
Thanks Greg,
Indeed I wanted msg.speakBlocking() but didn't know it existed.